app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class InvitationsController < ApplicationController
+	def new
+	end
+	def show
+    if !Member.find_by_email(params[:email]).present?
+      signup_params = '?website_id=' + params[:id] + '&email=' + params[:email] + '&token=' + params[:token]
+      redirect_to signup_url() + signup_params
+    else
+      WebsiteMember.where( invitation_token: params[:token] ).update_all( member_id: Member.find_by_email(params[:email]).id )
```

Comment:
cam ciudat totusi ca faci asta aici si nu in api desi faci request la api din ce am vazut .... cam ciudat trebuie sa imi explici te rog